### DISCUSSION TITLE
Restricting a kit

### DISCUSSION THREAD

**Discussion by scu-koverton** [FortAwesome/Font-Awesome#19475] [Q&A] [answered]
HI, We want to build a new kit for our new website. BUT we only want the think version is in the kit and only a selected amount of icons. Is this possible? Anyone done similar?

Thanks

**Comment by sensibleworld** [comment, marked as answer]
Hey there! We're working to become more active in discussions, and realized this one was left hanging! So, you've likely moved on... but I did want to say that yes, with a Pro account, you can restrict a Kit to specific domains, and only include certain styles, or even subset it to only include certain icons.
